packages/eventual-send/src/E.js

```diff
@@ -302,26 +302,90 @@ export default makeE;
  * `Promise<T>` or `T` itself.
  */
 
+/**
+ * A type that doesn't assume its parameter is local, but is satisfied with both
+ * local and remote references. It accepts both near and marshalled references
+ * that were returned from `Remotable` or `Far`.
+ *
+ * @template Primary The type of the primary reference.
+ * @template [Local=DataOnly<Primary>] The local properties of the object.
+ * @typedef {Primary | RemotableBrand<Local, Primary>} Remote
```

Comment:
If this mean "local or remote" as it seems to, "Remote" seems a strange name. Remotable already seems to mean what you want, with the difference being that your remote includes non-remotable local objects that also implement the Primary API. Is that what you intend?

Reply:
With `Remote` and `ERemote` gone, resolving.